{"name": "Проверка ошибки при несуществующем номера заказа", "status": "passed", "steps": [{"name": "Получение заказа", "status": "passed", "parameters": [{"name": "track_id", "value": "'999888777'"}], "start": 1744280280515, "stop": 1744280280611}], "start": 1744280280515, "stop": 1744280280611, "uuid": "f3cd11c4-3793-473d-bd20-f721849c6501", "historyId": "47a18b9e3b32995712fbddfdd08ba310", "testCaseId": "47a18b9e3b32995712fbddfdd08ba310", "fullName": "tests.tests_orders.test_get_order.TestOrderList#test_get_order_error_with_nonexistent_id", "labels": [{"name": "parentSuite", "value": "tests.tests_orders"}, {"name": "suite", "value": "test_get_order"}, {"name": "subSuite", "value": "TestOrderList"}, {"name": "host", "value": "Evgeny"}, {"name": "thread", "value": "16148-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_orders.test_get_order"}]}